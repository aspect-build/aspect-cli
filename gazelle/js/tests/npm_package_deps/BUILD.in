# aspect:js_files **/*.{js,ts}
# gazelle:js_npm_package_target_name pkg
# gazelle:js_project_naming_convention tsc
# gazelle:js_npm_package enabled
