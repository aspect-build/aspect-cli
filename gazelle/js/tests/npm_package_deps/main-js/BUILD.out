load("@aspect_rules_js//js:defs.bzl", "js_library")
load("@aspect_rules_js//npm:defs.bzl", "npm_package")
load("@npm//:defs.bzl", "npm_link_all_packages")

npm_link_all_packages(name = "node_modules")

js_library(
    name = "tsc",
    srcs = ["start.js"],
)

npm_package(
    name = "pkg",
    srcs = [
        "package.json",
        ":tsc",
    ],
    visibility = ["//visibility:public"],
)
